.github/workflows/release.yml

```diff
@@ -35,21 +35,13 @@ jobs:
         ### end manpage generation
 
       - name: Build RPM package for EL7
-        id: rpm_build_el7
-        uses: oamg/rpmbuild@el7
-        with:
-          spec_path: "packaging/convert2rhel.spec"
+        run: |
+          make rpm7
```

Comment:
Why not turning this into a matrix and simplifying even further?

Reply:
Given the automate manpage generation that @Andrew-ang9 is going to be working on I figured it's best to leave it for that. But yeah I'll just simplify it now (and also enable el9 builds) then Andrew can focus on the manpage generation commit in pre-commit

Reply:
Done